src/kv/kv.h

```diff
@@ -995,8 +1013,8 @@ namespace kv
 
       version = c.value();
 
-      const std::vector<uint8_t> data = serialise();
-      if (!data.size())
+      SerialisedMaps data = serialise();
```

Comment:
`const auto`?

Reply:
`auto` yes, `const` no since we are doing a `move` on the `data` a few lines down and if it is `const` it will invoke the copy ctor instead of the move one.